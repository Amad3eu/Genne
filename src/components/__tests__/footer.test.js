import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from '../footer';

describe('Footer component', () => {
  const component = shallow(<Footer />);

  it('renders correctly', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
