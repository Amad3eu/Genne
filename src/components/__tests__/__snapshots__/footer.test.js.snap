// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<div
  className="bg-gray-100 p-4 flex flex-col justify-center items-center shadow-inner mt-2"
>
  <div
    className="w-full flex flex-col sm:flex-row justify-evenly py-2"
  >
    <div
      className="sm:ml-0 sm:mr-6 order-last sm:order-none flex"
    >
      <h1
        className="text-base font-bold font-title text-xl sm:text-2xl mt-3 sm:mt-0"
      >
        <div
          className="flex sm:flex-col items-start mb-3 sm:mb-0"
        >
          <img
            alt="github profile markdown generator logo"
            className="hidden sm:block h-24"
            src="test-file-stub"
          />
          <div
            className="mr-2 sm:mr-0"
          >
            GitHub Profile 
            <img
              alt="github profile markdown generator logo"
              className="inline sm:hidden h-12"
              src="test-file-stub"
            />
            <span
              className="block sm:inline"
            >
              README Generator
            </span>
          </div>
        </div>
      </h1>
    </div>
    <div
      className="text-xl sm:text-base font-light sm:font-normal"
    >
      <div
        className="font-title font-bold mb-4 sm:mb-2"
      >
        <strong>
          Pages
        </strong>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <mockConstructor
          activeStyle={
            Object {
              "color": "#002ead",
            }
          }
          to="/addons"
        >
          Addons
        </mockConstructor>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <mockConstructor
          activeStyle={
            Object {
              "color": "#002ead",
            }
          }
          to="/support"
        >
          Support
        </mockConstructor>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <mockConstructor
          activeStyle={
            Object {
              "color": "#002ead",
            }
          }
          to="/about"
        >
          About
        </mockConstructor>
      </div>
    </div>
    <div
      className="text-xl sm:text-base font-light sm:font-normal"
    >
      <div
        className="font-title font-bold my-4 sm:my-0 sm:mb-2"
      >
        <strong>
          More
        </strong>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <a
          aria-label="Github rahuldkjain/github-profile-readme-generator"
          href="https://github.com/rahuldkjain/github-profile-readme-generator"
          target="blank"
        >
          Github
        </a>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <a
          aria-label="Releases on Github rahuldkjain/github-profile-readme-generator"
          href="https://github.com/rahuldkjain/github-profile-readme-generator/releases"
          target="blank"
        >
          Releases
        </a>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <a
          aria-label="Issues in rahuldkjain/github-profile-readme-generator"
          href="https://github.com/rahuldkjain/github-profile-readme-generator/issues"
          target="blank"
        >
          Issues
        </a>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <a
          aria-label="Pull Requests in rahuldkjain/github-profile-readme-generator"
          href="https://github.com/rahuldkjain/github-profile-readme-generator/pulls"
          target="blank"
        >
          Pull Requests
        </a>
      </div>
    </div>
    <div>
      <div
        className="font-title font-bold text-xl sm:text-base my-4 sm:my-0 sm:mb-2"
      >
        <strong>
          Join Community
        </strong>
      </div>
      <div
        className="ml-2 sm:ml-0"
      >
        <a
          aria-label="Discord of the community"
          href="https://discord.gg/HHMs7Eg"
          target="blank"
        >
          <img
            alt="Discord of the community"
            className="h-12"
            src="test-file-stub"
          />
        </a>
      </div>
    </div>
  </div>
  <div
    className="py-2 mt-2"
  >
    Developed in India
     
    <span
      aria-label="india"
      role="img"
    >
       
      ðŸ‡®ðŸ‡³
    </span>
  </div>
</div>
`;
